riot-lorawan/downlink/README.md

```diff
@@ -0,0 +1,29 @@
+# Downlinks from the LoRaWAN network
+
+## **Goal**
+Write an application that blinks a LED when it receives a message from the
```

Comment:
Maybe just print the message: the participants won't see the hardware.
Adding cameras per nodes in iot-lab is planned as future improvement but it's not available yet ;)

Reply:
ah, ofc! I will fix that